### ISSUE TITLE
Fortress aka Dwarven Halls feedback

### ISSUE THREAD

**Issue by yrmvgh** [jeremygurr/dcssca#328]
Thoughts so far:
1. Very pretty tileset, I'm glad you resurrected it
2. doesn't feel like a fortress, so the old name might be better. To be a fortress, it needs to have an "inside" and an "outside" whether that entrance is literally in a forest. or just in the dungeon, or via a small network of cave tunnels. This is why gnoll fortress actually feels like a (very tiny) fortress (in a weird location).
3. This place seems like a viable alternative for characters who are the right **level** to do the ending of Elf 3 or Hall of Blades, but lack the proper **resistances**.
   - In other words: "_I just cleared Elf 1 and 2, now what?_" If you have sufficient resistances or can spam Silence to deal with elven casters, do the **elf 3 end vault**. If your AC and EV are great, do **dwarf**. If your character has both resistances and defenses, then you can also do **Hall of Blades**.
4. DD zerkers who can summon trogbuddies are very scary. If you have the "perfect hallway choke point", the summoned angry troll will pop up behind you and mangle your back. (I think this is cool.)

What does everyone else think?

**Comment by Sandman25DCSS** [comment]
I like the branch except tiles of some monsters are identical to Jorgrun's tile so I have to carefully check them to make sure there is no Jorgrun. Though the branch is probably too easy and I got a cloak with rN+, it was not an artifact and the ego is impossible for cloak in vanilla crawl.

**Comment by yrmvgh** [comment]
Except for the ^trog guys and a Nikolai that spawned here, I would agree that most of the enemies are on the easy side.

Though ghoul players should note that quite a few Dispel Undead casters exist in this branch - @SleepySteel

**Comment by jeremygurr** [comment]
Ultimately I want this to feel like a later game form of a bailey. Maybe you could have an outside and inside, with dwarves firing crossbows at you while you try and cross the bridge, or maybe you could cross the moat if you can fly or swim and use a back door entrance. 

I'd like to move elf to be within forest, and just have fortress at the end of orc. It's meant to be a much greater challenge for those players who found orc maybe a little too easy, and would like to kick it up a few notches, for the promise of greater reward. Maybe there's a large treasure room, mostly focussed on gold, but with a few nice randart jewelry lying around also. The idea is if you can break into the fortress, you could find enough gold to pretty much buy any item in a store, or beef up your gozag character considerably. 

So there is obviously some work to do here. It's likely that some of the monsters are too easy because they may have lost a spell or ability they used to have, but no longer exists in the current version of crawl, and nothing has been given them to replace it. Another reason some would be too easy is the way monster difficulty was recorded is different now, and my guesses as to how they should translate across could easily be off by quite a bit, making monsters have less hp and damage than they should have. 

It's really not too hard to make vaults. Basic vaults don't require any programming skill. There is good documentation in the docs folder. And tweaking monster stats is fairly easy too. I will get to all of this eventually, but if someone else were to contribute, it would come much faster. Another way that people could contribute is to suggest abilities or adjustments to monsters. If there are existing abilities or spells that would make sense for a monster to have, they can be added quickly and easily in most cases. 

I'd like it eventually to be about the size of orc and elf. Maybe 3 or 4 floors. But it should be noticably  harder than elf and even slime. Maybe it could have a nice boss like slime does at the bottom floor. Maybe two bosses, a treasure room guardian and an executioner (with a sweet randart executioner axe of course). 

Or maybe it would be more appropriate for the treasure of a fortress to focus on weapons or armour. I would like it somewhat distinct from tomb and elf treasure floors if possible.

**Comment by yrmvgh** [comment]
Because of the ^trog creatures, it's not a cakewalk - note that Sandman indicated in another issue that he hasn't tried spellcasting. I think this vault is significantly more dangerous to the lightly armored such as my djinn, though currently that danger is mostly all tied up in one type of monster. I'll throw some new stuff on some of the other monsters, we'll see what sticks.

"Bailey feel" is a helpful target to know that we're aiming for!

**Comment by jeremygurr** [comment]
Yeah that makes sense that fortress would be easier for melee characters and harder for spellcasters, in the same way that elf should be harder for melee and easier for spellcasters.
